src/roshnikolhe/Assignment2.java

```diff
@@ -0,0 +1,98 @@
+/*
+ Assignment - 2 : 18th Nov'2023 [30 mins]
+
+1. Launch Chrome Browser
+2. Navigate to automationbykrishna.com
+3. Click on Registration link
+4. Enter username
+5. Enter password having length > 8 characters
+6. Click on Login button
+7. Verify alter message should be "Success!"
+8. clear username and password
+9. Enter username
+10.Enter password having length < 8 characters
+11. Click on Login button
+12. Verify alter message should be "Failed! please enter strong password"
+ */
+
+package roshnikolhe;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Wait;
+
+public class Assignment2 {
+	
+	public static void main(String[] args) {
+		System.out.println("Step1. Launch Chrome Browser");
+		//String SetProperty = System.setProperty("webdriver.chrome.driver", "D:\\\\Technocredits\\\\Projects\\\\IDE\\\\Sep23Selenium\\\\ChromeDriver\\\\chromedriver.exe");
```

Comment:
this path should be relative, what if I want to execute the script on my local machine